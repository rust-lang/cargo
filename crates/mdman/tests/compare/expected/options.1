'\" t
.TH "MY\-COMMAND" "1"
.nh
.ad l
.ss \n[.ss] 0
.SH "NAME"
my\-command \- A brief description
.SH "SYNOPSIS"
\fBmy\-command\fR [\fB\-\-abc\fR | \fB\-\-xyz\fR] \fIname\fR
.br
\fBmy\-command\fR [\fB\-f\fR \fIfile\fR]
.br
\fBmy\-command\fR (\fB\-m\fR | \fB\-M\fR) [\fIoldbranch\fR] \fInewbranch\fR
.br
\fBmy\-command\fR (\fB\-d\fR | \fB\-D\fR) [\fB\-r\fR] \fIbranchname\fR\[u2026]
.SH "DESCRIPTION"
A description of the command.
.sp
.RS 4
\h'-04'\(bu\h'+03'One
.sp
.RS 4
\h'-04'\(bu\h'+03'Sub one
.RE
.sp
.RS 4
\h'-04'\(bu\h'+03'Sub two
.RE
.RE
.sp
.RS 4
\h'-04'\(bu\h'+03'Two
.RE
.sp
.RS 4
\h'-04'\(bu\h'+03'Three
.RE
.SH "OPTIONS"
.SS "Command options"
.sp
\fB\-\-foo\-bar\fR
.RS 4
Demo \fIemphasis\fR, \fBstrong\fR, ~~strike~~
.RE
.sp
\fB\-p\fR \fIspec\fR, 
\fB\-\-package\fR \fIspec\fR
.RS 4
This has multiple flags.
.RE
.sp
\fInamed\-arg\[u2026]\fR
.RS 4
A named argument.
.RE
.SS "Common Options"
.sp
\fB@\fR\fIfilename\fR
.RS 4
Load from filename.
.RE
.sp
\fB\-\-foo\fR [\fIbar\fR]
.RS 4
Flag with optional value.
.RE
.sp
\fB\-\-foo\fR[\fB=\fR\fIbar\fR]
.RS 4
Alternate syntax for optional value (with required = for disambiguation).
.RE
.sp
\fB\-\-split\-block\fR
.RS 4
An option where the description has a \fBblock statement that is split across multiple lines\fR
.RE
.SH "EXAMPLES"
.sp
.RS 4
\h'-04' 1.\h'+01'An example
.sp
.RS 4
.nf
my\-command \-\-abc
.fi
.RE
.RE
.sp
.RS 4
\h'-04' 2.\h'+01'Another example
.sp
.RS 4
.nf
my\-command \-\-xyz
.fi
.RE
.RE
.SH "SEE ALSO"
\fBother\-command\fR(1) \fBabc\fR(7)
