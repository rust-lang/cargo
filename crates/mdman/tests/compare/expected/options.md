# my-command(1)

## NAME

my-command - A brief description

## SYNOPSIS

`my-command` [`--abc` | `--xyz`] _name_\
`my-command` [`-f` _file_]\
`my-command` (`-m` | `-M`) [_oldbranch_] _newbranch_\
`my-command` (`-d` | `-D`) [`-r`] _branchname_...

## DESCRIPTION

A description of the command.

* One
    * Sub one
    * Sub two
* Two
* Three


## OPTIONS

### Command options

<dl>

<dt class="option-term" id="option-options---foo-bar"><a class="option-anchor" href="#option-options---foo-bar"></a><code>--foo-bar</code></dt>
<dd class="option-desc">Demo <em>emphasis</em>, <strong>strong</strong>, <del>strike</del></dd>


<dt class="option-term" id="option-options--p"><a class="option-anchor" href="#option-options--p"></a><code>-p</code> <em>spec</em></dt>
<dt class="option-term" id="option-options---package"><a class="option-anchor" href="#option-options---package"></a><code>--package</code> <em>spec</em></dt>
<dd class="option-desc">This has multiple flags.</dd>


<dt class="option-term" id="option-options-named-arg…"><a class="option-anchor" href="#option-options-named-arg…"></a><em>named-arg…</em></dt>
<dd class="option-desc">A named argument.</dd>


</dl>

### Common Options

<dl>
<dt class="option-term" id="option-options-@filename"><a class="option-anchor" href="#option-options-@filename"></a><code>@</code><em>filename</em></dt>
<dd class="option-desc">Load from filename.</dd>


<dt class="option-term" id="option-options---foo"><a class="option-anchor" href="#option-options---foo"></a><code>--foo</code> [<em>bar</em>]</dt>
<dd class="option-desc">Flag with optional value.</dd>


<dt class="option-term" id="option-options---foo[=bar]"><a class="option-anchor" href="#option-options---foo[=bar]"></a><code>--foo</code>[<code>=</code><em>bar</em>]</dt>
<dd class="option-desc">Alternate syntax for optional value (with required = for disambiguation).</dd>


<dt class="option-term" id="option-options---split-block"><a class="option-anchor" href="#option-options---split-block"></a><code>--split-block</code></dt>
<dd class="option-desc">An option where the description has a <code>block statement that is split across multiple lines</code></dd>


</dl>


## EXAMPLES

1. An example

   ```
   my-command --abc
   ```

1. Another example

       my-command --xyz

## SEE ALSO
[other-command(1)](other-command.html) [abc(7)](abc.html)
