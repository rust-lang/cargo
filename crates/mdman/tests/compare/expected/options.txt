MY-COMMAND(1)

NAME
       my-command - A brief description

SYNOPSIS
       my-command [--abc | --xyz] name
       my-command [-f file]
       my-command (-m | -M) [oldbranch] newbranch
       my-command (-d | -D) [-r] branchname…

DESCRIPTION
       A description of the command.

       o  One
          o  Sub one

          o  Sub two

       o  Two

       o  Three

OPTIONS
   Command options
       --foo-bar
           Demo emphasis, strong, ~~strike~~

       -p spec, --package spec
           This has multiple flags.

       named-arg…
           A named argument.

   Common Options
       @filename
           Load from filename.

       --foo [bar]
           Flag with optional value.

       --foo[=bar]
           Alternate syntax for optional value (with required = for
           disambiguation).

       --split-block
           An option where the description has a block statement that is split
           across multiple lines

EXAMPLES
       1. An example

              my-command --abc

       2. Another example

              my-command --xyz

SEE ALSO
       other-command(1) abc(7)

